interfaces/builtin/process_control.go

```diff
@@ -46,6 +46,8 @@ capability kill,
 signal (send),
 /{,usr/}bin/kill ixr,
 /{,usr/}bin/pkill ixr,
+
+@{PROC}/self/coredump_filter wr,
```

Comment:
this is interface is mainly about controlling also other processes, so self doesn't entirely make sense here

Reply:
also `self` is a symlink but AppArmor only deals with real files - so this would instead need to be something like:

```
    @{PROC}/@{pid}/coredump_filter rw,
```

Reply:
the question is really whether it makes sense to give OpenSearch process_control as it allows to affect other processes?? otherwise we need to add something more specific somewhere (either new or old, although I don't see any current interface that would be a good candidate) cc @alexmurray 

in the context of this interface we would probably add:

 @{PROC}/[0-9]*/coredump_filter rw,

Reply:
Opensearch already needs process_control for the "sys_resource" capability.

Reply:
As I suggested above, can this please use `@{pid}` instead of `[0-9]*` - since eventually AppArmor is supposed to understand that `@{pid}` will refer to just the current processes pid rather than allowing all processes (which however is what `@{pid}` currently is used for)?